spec/controllers/users_controller_spec.rb

```diff
@@ -2,8 +2,48 @@
 
 describe UsersController, type: :controller do
 
-  let!(:admin) {create(:user, :admin)}
-  let!(:hacker) {create(:user)}
+  let!(:admin) {create(:user, :admin, name: 'Admin')}
+  let!(:hacker) {create(:user, name: 'Hacker')}
+
+  describe 'GET index' do
+
+    context 'default index list' do
+      before :each do
+        allow(controller).to receive(:current_user) { admin }
+      end
+
+      it "returns all users" do
+        smith = create(:user, name: 'Jane Smith')
+        jones = create(:user, name: 'Frank Jones')
+        get :index
+        expect(assigns(:users)).not_to be_nil
+        #expect(assigns(:users)).to contain([smith, jones])
```

Comment:
`        expect(assigns(:users)).to include(smith, jones)`

just worked for me, you don't want it to contain an array, just the elements